build-packages.cmd

```diff
@@ -7,6 +7,10 @@ echo Running build-packages.cmd %* > %packagesLog%
 if /I [%1] == [/?] goto Usage
 if /I [%1] == [/help] goto Usage
 
+REM ensure that msbuild is available
+echo Running init-tools.cmd
+call %~dp0init-tools.cmd
```

Comment:
init-tools has its own log file, but we can add to packagesLog that we are calling init-tools